script/images/imagemin.js

```diff
@@ -1,18 +1,9 @@
 var execCommand = require('../helper/exec-command');
 var helper = require('../helper');
-var symlinkOrCopySync = require('symlink-or-copy').sync;
 var buildFolder = helper.getBuildRootFolder();
-var isProduction = helper.isProductionBuild();
 
 /**
- * Prod: minify and copy images
- * Dev: symlink images folder, or copy images if symlink is not available
+ * minify and copy images
  */
-if(isProduction) {
```

Comment:
Can you double check images are not minified in their original source but only when moved to the build folder?

Reply:
Why we can't use the same approach that we use on prod? Do we need images not minified for dev env?

Reply:
I remember I had issues with minification. In particular minification was done on images on their location in _src_ (resulting in a git change) instead of first copying them to _build_ and then apply minification.

Can you check the script doesn't minify images in _src_ but only in _build_?

Reply:
Checked by adding a new image: no changes on src folder